gdk4-win32/Cargo.toml

```diff
@@ -0,0 +1,63 @@
+[package]
+authors = ["The gtk-rs Project Developers"]
+description = "Rust bindings of the GDK4 Win32 library"
+documentation = "https://gtk-rs.org/gtk4-rs/stable/latest/docs/gdk4_win32/"
+edition = "2021"
+homepage = "https://gtk-rs.org/"
+keywords = ["gdk4", "gdk4-win32", "gtk-rs", "gnome", "GUI"]
+license = "MIT"
+name = "gdk4-win32"
+readme = "README.md"
+repository = "https://github.com/gtk-rs/gtk4-rs"
+version = "0.5.0"
+rust-version = "1.63"
+build = "build.rs"
+
+[package.metadata.docs.rs]
+features = ["dox"]
+
+[package.metadata.system-deps.gtk4_win32]
+name = "gtk4-win32"
+version = "4"
+
+[features]
+v4_4 = ["ffi/v4_4"]
+dox = ["ffi/dox", "gdk/dox", "gio/dox", "glib/dox", "egl", "win32"]
+egl = ["khronos-egl"]
+win32 = ["windows"]
+
+[dependencies]
+libc = "0.2"
+khronos-egl = {version = "4.1.0", optional = true}
+windows = { version = "0.41.0", features = [
+    "Win32_Foundation",
+    "Win32_UI_WindowsAndMessaging",
+], optional = true }
+
+[dependencies.ffi]
+path = "./sys"
+package = "gdk4-win32-sys"
+version = "0.5.0"
+
+[dependencies.gdk]
+path = "../gdk4"
+package = "gdk4"
+version = "0.5.0"
+
+[dependencies.gio]
+git = "https://github.com/gtk-rs/gtk-rs-core"
+version = "0.17"
```

Comment:
should be set to 0.16 as well as adding a `branch = "0.16"` for all the gtk-rs-core dependencies